README.md

```diff
@@ -101,12 +101,12 @@ extension UIImage {
     case BigPear = "Big_Pear"
     case StopButtonEnabled = "stop.button.enabled"
 
-    var image: UIImage {
-      return UIImage(named: self.rawValue)!
+    var image: UIImage? {
+      return UIImage(named: self.rawValue)
     }
   }
 
-  convenience init!(asset: Asset) {
+  convenience init?(asset: Asset) {
```

Comment:
Oh, that's not what I was thinking about at all, sorry…

Given that the enums are auto-generated by SwiftGen each time you build (as you generally setup SwiftGen in a Script Build Phase), that is **very unlikely** that the init would fail here (and same for the computed property `image: UIImage`). So making them plain optionals would be a pain to use then because you'll have to unwrap them at call site, whereas you know that they should never be `nil` (because SwiftGen will ensure they are in sync with the assets catalog) and if they are, that's a programmer error (which should generate a `fatalError`).

So making this optional here just deports the unwrapping at call site, which would then be a pain to unwrap each time for nothing.

What I was thinking about was more along the lines of this:

```swift
convenience init(asset: Asset) {
  self.init(named: asset.rawValue)
  guard self != nil else {
    fatalError(
      "You used an Asset enum which doesn't match an existing Image. \n"
      + "Be sure to run SwiftGen again to ensure the enum and the Assets Catalog are in sync"
    )
  }
}
```

You can find some inspiration [on how I do this in my Reusable pod on those lines](https://github.com/AliSoftware/Reusable/blob/master/Sources/Reusable.swift#L63-L70).